src/mongo_client.ts

```diff
@@ -32,6 +32,32 @@ import type { SrvPoller } from './sdam/srv_polling';
 import type { Connection } from './cmap/connection';
 import type { LEGAL_TLS_SOCKET_OPTIONS, LEGAL_TCP_SOCKET_OPTIONS } from './cmap/connect';
 
+/** @public */
+export const LogLevel = {
+  error: 'error',
+  warn: 'warn',
+  info: 'info',
+  debug: 'debug'
+} as const;
+
+/** @public */
+export const ServerApiVersion = {
```

Comment:
The spec calls for this to be an enum if the language supports it, but we avoid the use of TypeScript enums so I used the same pattern as `LogLevel` here, does that seem appropriate?

Reply:
Seems good to me. Although LogLevel is now defined in logger.ts maybe there was a rebase issue here?

Reply:
Oops yeah that's likely, fixed!